Library/Homebrew/cask/audit.rb

```diff
@@ -545,6 +545,38 @@ def check_livecheck_version
       false
     end
 
+    def check_livecheck_min_os
+      return unless online?
+      return unless cask.livecheckable?
+      return unless cask.livecheck.strategy == :sparkle
+
+      out, _, status = curl_output(cask.livecheck.url)
```

Comment:
```suggestion
      out, _, status = curl_output("--fail", "--silent", "--location", cask.livecheck.url)
```

So that it will detect non-200 codes, follow redirects etc.